src/Main/TodoItem.tsx

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable jsx-a11y/no-autofocus */
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Avoid disabling eslint rules unless necessary. Remove this eslint disable comment or explain why it's needed.